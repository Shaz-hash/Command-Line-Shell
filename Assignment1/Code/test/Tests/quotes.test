find .. -type f -name "*.c" -o -name "*.h" | xargs wc -l | sort -n && find .. -type f -name "*.c" -o -name "*.h" | xargs wc -l | tail -n 1 | awk '{print $1}' | xargs echo "LOC: "