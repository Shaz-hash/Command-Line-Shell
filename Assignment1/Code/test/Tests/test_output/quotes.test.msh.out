   27 ../include/utils.h
  101 ../include/log.h
 2055 ../src/main.c
 2183 total
